### PR TITLE
Nitpick fixes

### PR DESCRIPTION
**Pull request by pacrob**
## What was wrong?

Update of PR #79 
Credit to @AndreMiras 
"Nothing wrong simply a couple of styling/typo fixes."

## How was it fixed?

"Made the non-functional changes."

### To-Do

[//]: # (See: https://eth-account.readthedocs.io/en/latest/contributing.html#pull-requests)
- [x] Add entry to the [release notes](https://github.com/ethereum/eth-account/blob/master/newsfragments/README.md)

#### Cute Animal Picture

image

### CODE REVIEW

File: newsfragments/79.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+clean up some code style and spelling
```
Comment (fselmo) [inline newsfragments/79.misc.rst:1]: this should be named `179.misc.rst` 👀

Comment (fselmo) [reply newsfragments/79.misc.rst:1]: oh weird coincidence... hah nevermind

**Review by fselmo — APPROVED** [review]
I'm good with it 👍

**Comment by pacrob** [comment]
closes #79

**Pull request merged**